girder_worker/__init__.py

```diff
@@ -6,3 +7,24 @@
 _cfgs = ('worker.dist.cfg', 'worker.local.cfg')
 config = SafeConfigParser(os.environ)
 config.read([os.path.join(PACKAGE_DIR, f) for f in _cfgs])
+
+
+class GirderWorkerPluginABC(object):
```

Comment:
yup,  i'm actually working on an example plugin currently along with documentation.  Should be able to push in a little bit.